components/pkg-export-helm/src/main.rs

```diff
@@ -96,6 +96,18 @@ fn cli<'a, 'b>() -> clap::App<'a, 'b> {
                 .long("desc")
                 .help("A single-sentence description"),
         )
+        .arg(
+            Arg::with_name("OPERATOR_VERSION")
+                .value_name("OPERATOR_VERSION")
+                .long("operator-version")
+                .validator(valid_version)
+                .help("Version of the Habitat operator to set as dependency")
+                .default_value(deps::DEFAULT_OPERATOR_VERSION),
+        )
+        .arg(Arg::with_name("DOWNLOAD_DEPS").long("download-deps").help(
+            "Whether to download dependencies. The Kubernetes Habitat Operator is the only \
+             dependancy currently. (default: no)",
```

Comment:
Can we just use `.default_value(false)` and have that do the right thing?

Reply:
`default_value` expects an `&str` so that wont' work.

Reply:
_That's_ unfortunate 😦

Reply:
yeah :(